### ISSUE TITLE
Issue related to TF version in DeepVariant 

### ISSUE THREAD

**Issue by tahashmi** [google/deepvariant#555]
Is this issue related to TF version?
Any help to fix this issue? Thanks.
```
(base) [tahmad@gcn35 tests]$  BIN_VERSION="1.4.0"
(base) [tahmad@gcn35 tests]$ singularity run --nv -B  /usr/lib/locale/ docker://google/deepvariant:${BIN_VERSION}-gpu  /opt/deepvariant/bin/run_deepvariant   --model_type PACBIO     --ref reference/GRCh38_no_alt_analysis_set.fasta
     --reads input/HG003.GRCh38.chr20.pFDA_truthv2.bam     --output_vcf deepvariant_output/output.vcf.gz     --num_shards $(nproc)     --regions chr20
INFO:    Using cached SIF image
2022-08-20 12:59:52.389461: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
Traceback (most recent call last):
  File "/opt/deepvariant/bin/run_deepvariant.py", line 48, in <module>
    import tensorflow as tf
  File "/home/tahmad/.local/lib/python3.8/site-packages/tensorflow/__init__.py", line 444, in <module>
    _ll.load_library(_main_dir)
  File "/home/tahmad/.local/lib/python3.8/site-packages/tensorflow/python/framework/load_library.py", line 154, in load_library
    py_tf.TF_LoadLibrary(lib)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/local/lib/python3.8/dist-packages/tensorflow/core/kernels/libtfkernel_sobol_op.so: undefined symbol: _ZN10tensorflow14kernel_factory17OpKernelRegistrar12InitInternalEPKNS_9KernelDefEN4absl12lts_2021032411string_viewESt10unique_ptrINS0_15OpKernelFactoryESt14default_deleteIS9_EE
```

**Comment by kishwarshafin** [comment]
Hi @tahashmi ,

It does look like a TensorFlow issue as described here: https://github.com/tensorflow/text/issues/385

You can check the tensorflow version of DeepVariant this way:
```bash
singularity run --nv -B  /usr/lib/locale/ docker://google/deepvariant:${BIN_VERSION}-gpu  python3 -c 'import tensorflow as tf; print(tf.__version__)'
```
And then please try to install the tensorflow version locally to see if the error gets fixed.

**Comment by tahashmi** [comment]
Thanks @kishwarshafin . 
But I think I am not even able to import TF in Python environment through DV container. 

```
(base) [tahmad@gcn4 ~]$ singularity run --nv -B  /usr/lib/locale/ docker://google/deepvariant:${BIN_VERSION}-gpu  python3
INFO:    Using cached SIF image
Python 3.8.10 (default, Mar 15 2022, 12:22:08)
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
2022-08-28 09:48:47.608744: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tahmad/.local/lib/python3.8/site-packages/tensorflow/__init__.py", line 444, in <module>
    _ll.load_library(_main_dir)
  File "/home/tahmad/.local/lib/python3.8/site-packages/tensorflow/python/framework/load_library.py", line 154, in load_library
    py_tf.TF_LoadLibrary(lib)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/local/lib/python3.8/dist-packages/tensorflow/core/kernels/libtfkernel_sobol_op.so: undefined symbol: _ZN10tensorflow14kernel_factory17OpKernelRegistrar12InitInternalEPKNS_9KernelDefEN4absl12lts_2021032411string_viewESt10unique_ptrINS0_15OpKernelFactoryESt14default_deleteIS9_EE
```

on my local system, I have TF 2.5.2
```
>>> import tensorflow as tf
2022-08-28 09:51:55.901400: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
>>> print(tf.__version__)
2.5.2
```

**Comment by tahashmi** [comment]
For the time being, this issue has been resolved by updating my local TF to 2.7.0 which is used in DV container.
`pip install -U tensorflow==2.7.0`
